setup.py

```diff
@@ -6,7 +6,7 @@
     name='shub',
     version='2.7.0',
     packages=find_packages(exclude=('tests', 'tests.*')),
-    url='http://doc.scrapinghub.com/shub.html',
+    url='https://shub.readthedocs.io',
```

Comment:
Maybe we can have one central place where we define all the docs links? E.g. in `shub/__init__.py`, and then always import from there?